errbot/core_plugins/chatRoom.py

```diff
@@ -176,6 +175,7 @@ def room_occupants(self, message, args):
         Examples (IRC):
         !room occupants #example-room #another-example-room
         """
+        print("args: ", args)
```

Comment:
Think you accidentally included this in your commit here. :smiley_cat:

Reply:
:/ sorry, my bad. Will fix right away.

Reply:
No worries, been there, done that. :smile: